include/rdma/fabric.h

```diff
@@ -336,6 +351,7 @@ struct fi_tx_attr {
 	size_t			size;
 	size_t			iov_limit;
 	size_t			rma_iov_limit;
+	uint32_t		tclass;
```

Comment:
I would still define this as an EP attribute only, not as separate tx rx attributes.

Reply:
I think this still needs to be present to support traffic classes by a specific tx context. This applies to the scalable endpoint case correct?

Reply:
Hmm... you're correct.  It's needed for the SEP case.  I agree that it makes the most sense for each Tx context to have its own TC.

That brings up defining the shared Tx context case as well.  I would have a shared Tx define a default setting, but allow each bound EP to set their own.

I would still remove it from the Rx attribute.

Reply:
> I would still remove it from the Rx attribute.

I thiink that's gone already.